### DISCUSSION TITLE
How to format cells

### DISCUSSION THREAD

**Discussion by robertosanval** [spatie/simple-excel#154] [Q&A]
I need help finding how to format cells or a whole column as Number or DateTime.

Any ideas?

**Comment by tylerwiegand** [comment]
I recently had this issue and believe this is how it ought to be done in the context of this Spatie package:

```php
$row = (new Row([
            Cell::fromValue('Regular Cell'),
            Cell::fromValue('Styled Cell', (new Style)->setFormat('[hh]:mm:ss'))
        ]));
$writer->addRow($row);
```

Hope that helps you or someone in the future
